Parser/Client/Browser.php

```diff
@@ -410,6 +411,7 @@ class Browser extends AbstractClientParser
         'MS' => 'Mobile Silk',
         'MK' => 'Mogok Browser',
         'M8' => 'Motorola Internet Browser',
+        '4M' => 'Motorola WebKit',
```

Comment:
I do not think this is a browser. Matomo already has MotorolaWebkit [here](https://github.com/matomo-org/device-detector/blob/cd2846d831b75f0a363701d634af3975731755eb/Tests/fixtures/smartphone-9.yml#L5091) and it's treated as Mobile Safari. But overall, I'd say it's most likely [this](https://github.com/matomo-org/device-detector/blob/cd2846d831b75f0a363701d634af3975731755eb/regexes/client/browsers.yml#L112). While Motorola did have own fork of WebKit, I just do not see them actually having a separate browser, it looks more like they called the _engine_ as _browser_.